lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,68 @@
+module ArPglogicalMigration
+  module ArPglogicalMigrationHelper
+    def self.migrations_column_present?
+      ActiveRecord::Base.connection.columns("miq_regions").any? { |c| c.name == "migrations_ran" }
+    end
+
+    def self.log_and_print(message)
+      if @current_message == message
+        print "."
+      else
+        Vmdb.rails_logger.info(message)
+        print message
+      end
+      @current_message = message
+    end
+
+    class HelperARClass < ActiveRecord::Base; end
+
+    def self.restart_subscription(s)
+      c = HelperARClass.establish_connection.connection
```

Comment:
>I'm not sure. I think there's a single pool per AR class. Plus I didn't want to mess with the "real" connection pool here.

>@jrafanie may have a better idea.

I like this way. 👍   We're doing very specific database work here unrelated to models and only need the database connection.  I'd rather not possibly infect other models' connection.